Remotion/Web/Core/Res/Themes/NovaViso/HTML/Common.css

```diff
@@ -31,6 +31,8 @@
   --remotion-themed-min-height: calc(var(--remotion-themed-line-height) + 2 * var(--remotion-themed-padding));
   --remotion-themed-multiline-min-height: calc(2 * var(--remotion-themed-line-height) + 2 * var(--remotion-themed-padding));
   --remotion-themed-control-width: 16em;
+  --default-icon-size: 16px;
+  --default-text-color: black;
```

Comment:
That's --color-standard-text with #000000. please add in line 21 before background.